ios/NotificationsHelperIOS10.swift

```diff
@@ -59,11 +60,14 @@ class NotificationsHelperIOS10: NSObject, UNUserNotificationCenterDelegate {
     }))
   }
   
-  private func showRichNotification(title: String, subtitle: String, body: String, url: String) {
+  private func showRichNotification(title: String, subtitle: String, body: String, image: String?, url: String) {
     let content = UNMutableNotificationContent()
     content.title = title
     content.subtitle = subtitle
     content.body = body
+    if let image: String = image! {
```

Comment:
It just looks weird that you redefine the variable just to check that it's defined.